MediaBrowser.Providers/Plugins/AudioDb/Plugin.cs

```diff
@@ -19,6 +19,9 @@ public class Plugin : BasePlugin<PluginConfiguration>, IHasWebPages
 
         public override string Description => "Get artist and album metadata or images from AudioDB.";
 
+        // TODO remove when plugin removed from server.
+        public override string ConfigurationFileName => "Jellyfin.Plugin.AudioDb.xml";
```

Comment:
Do any of these have existing configs? And does this change affect anything in that respect?

Reply:
I'm not sure where they were stored in 10.5, but as of 10.6 all of the configurations were reset and save to the same file.